### ISSUE TITLE
train 自动退出 没有报错信息

### ISSUE THREAD

**Issue by liuzimo** [PaddlePaddle/PaddleOCR#9724]
- 系统环境/System Environment：
- win10 python3.8
- 版本号/Paddle： 
- paddle-bfloat                0.1.7
paddledet                    2.5.0
paddlepaddle-gpu             2.4.1.post116

 PaddleOCR：2.6.0 
- 运行指令/Command Code：
- 
Microsoft Windows [版本 10.0.19044.1766]
(c) Microsoft Corporation。保留所有权利。

E:\Backup\Desktop\手写字体检测\PaddleOCR-2.6.0>conda activate py38

(py38) E:\Backup\Desktop\手写字体检测\PaddleOCR-2.6.0>python tools/train.py -c configs/rec/rec_r34_vd_none_bilstm_ctc.yml -o Global.pretrained_model=./rec_r34_vd_none_bilstm_ctc_v2.0_train/best_accuracy
[2023/04/14 13:45:43] ppocr INFO: Architecture :
[2023/04/14 13:45:43] ppocr INFO:     Backbone :
[2023/04/14 13:45:43] ppocr INFO:         layers : 34
[2023/04/14 13:45:43] ppocr INFO:         name : ResNet
[2023/04/14 13:45:43] ppocr INFO:     Head :
[2023/04/14 13:45:43] ppocr INFO:         fc_decay : 0
[2023/04/14 13:45:43] ppocr INFO:         name : CTCHead
[2023/04/14 13:45:43] ppocr INFO:     Neck :
[2023/04/14 13:45:43] ppocr INFO:         encoder_type : rnn
[2023/04/14 13:45:43] ppocr INFO:         hidden_size : 256
[2023/04/14 13:45:43] ppocr INFO:         name : SequenceEncoder
[2023/04/14 13:45:43] ppocr INFO:     Transform : None
[2023/04/14 13:45:43] ppocr INFO:     algorithm : CRNN
[2023/04/14 13:45:43] ppocr INFO:     model_type : rec
[2023/04/14 13:45:43] ppocr INFO: Eval :
[2023/04/14 13:45:43] ppocr INFO:     dataset :
[2023/04/14 13:45:43] ppocr INFO:         data_dir : ./data
[2023/04/14 13:45:43] ppocr INFO:         label_file_list : ['./data/rec_gt_test.txt']
[2023/04/14 13:45:43] ppocr INFO:         name : SimpleDataSet
[2023/04/14 13:45:43] ppocr INFO:         transforms :
[2023/04/14 13:45:43] ppocr INFO:             DecodeImage :
[2023/04/14 13:45:43] ppocr INFO:                 channel_first : False
[2023/04/14 13:45:43] ppocr INFO:                 img_mode : BGR
[2023/04/14 13:45:43] ppocr INFO:             CTCLabelEncode : None
[2023/04/14 13:45:43] ppocr INFO:             RecResizeImg :
[2023/04/14 13:45:43] ppocr INFO:                 image_shape : [3, 32, 100]
[2023/04/14 13:45:43] ppocr INFO:             KeepKeys :
[2023/04/14 13:45:43] ppocr INFO:                 keep_keys : ['image', 'label', 'length']
[2023/04/14 13:45:43] ppocr INFO:     loader :
[2023/04/14 13:45:43] ppocr INFO:         batch_size_per_card : 256
[2023/04/14 13:45:43] ppocr INFO:         drop_last : False
[2023/04/14 13:45:43] ppocr INFO:         num_workers : 1
[2023/04/14 13:45:43] ppocr INFO:         shuffle : False
[2023/04/14 13:45:43] ppocr INFO: Global :
[2023/04/14 13:45:43] ppocr INFO:     cal_metric_during_train : True
[2023/04/14 13:45:43] ppocr INFO:     character_dict_path : data/class.txt
[2023/04/14 13:45:43] ppocr INFO:     checkpoints : None
[2023/04/14 13:45:43] ppocr INFO:     distributed : False
[2023/04/14 13:45:43] ppocr INFO:     epoch_num : 72
[2023/04/14 13:45:43] ppocr INFO:     eval_batch_step : [0, 2000]
[2023/04/14 13:45:43] ppocr INFO:     infer_img : doc/imgs_words_en/word_10.png
[2023/04/14 13:45:43] ppocr INFO:     infer_mode : False
[2023/04/14 13:45:43] ppocr INFO:     log_smooth_window : 20
[2023/04/14 13:45:43] ppocr INFO:     max_text_length : 25
[2023/04/14 13:45:43] ppocr INFO:     pretrained_model : ./rec_r34_vd_none_bilstm_ctc_v2.0_train/best_accuracy
[2023/04/14 13:45:43] ppocr INFO:     print_batch_step : 10
[2023/04/14 13:45:43] ppocr INFO:     save_epoch_step : 3
[2023/04/14 13:45:43] ppocr INFO:     save_inference_dir : None
[2023/04/14 13:45:43] ppocr INFO:     save_model_dir : ./output/rec/r34_vd_none_bilstm_ctc/
[2023/04/14 13:45:43] ppocr INFO:     save_res_path : ./output/rec/predicts_r34_vd_none_bilstm_ctc.txt
[2023/04/14 13:45:43] ppocr INFO:     use_gpu : True
[2023/04/14 13:45:43] ppocr INFO:     use_space_char : False
[2023/04/14 13:45:43] ppocr INFO:     use_visualdl : False
[2023/04/14 13:45:43] ppocr INFO: Loss :
[2023/04/14 13:45:43] ppocr INFO:     name : CTCLoss
[2023/04/14 13:45:43] ppocr INFO: Metric :
[2023/04/14 13:45:43] ppocr INFO:     main_indicator : acc
[2023/04/14 13:45:43] ppocr INFO:     name : RecMetric
[2023/04/14 13:45:43] ppocr INFO: Optimizer :
[2023/04/14 13:45:43] ppocr INFO:     beta1 : 0.9
[2023/04/14 13:45:43] ppocr INFO:     beta2 : 0.999
[2023/04/14 13:45:43] ppocr INFO:     lr :
[2023/04/14 13:45:43] ppocr INFO:         learning_rate : 0.0005
[2023/04/14 13:45:43] ppocr INFO:     name : Adam
[2023/04/14 13:45:43] ppocr INFO:     regularizer :
[2023/04/14 13:45:43] ppocr INFO:         factor : 0
[2023/04/14 13:45:43] ppocr INFO:         name : L2
[2023/04/14 13:45:43] ppocr INFO: PostProcess :
[2023/04/14 13:45:43] ppocr INFO:     name : CTCLabelDecode
[2023/04/14 13:45:43] ppocr INFO: Train :
[2023/04/14 13:45:43] ppocr INFO:     dataset :
[2023/04/14 13:45:43] ppocr INFO:         data_dir : ./data
[2023/04/14 13:45:43] ppocr INFO:         label_file_list : ['./data/rec_gt_train.txt']
[2023/04/14 13:45:43] ppocr INFO:         name : SimpleDataSet
[2023/04/14 13:45:43] ppocr INFO:         transforms :
[2023/04/14 13:45:43] ppocr INFO:             DecodeImage :
[2023/04/14 13:45:43] ppocr INFO:                 channel_first : False
[2023/04/14 13:45:43] ppocr INFO:                 img_mode : BGR
[2023/04/14 13:45:43] ppocr INFO:             CTCLabelEncode : None
[2023/04/14 13:45:43] ppocr INFO:             RecResizeImg :
[2023/04/14 13:45:43] ppocr INFO:                 image_shape : [3, 32, 100]
[2023/04/14 13:45:43] ppocr INFO:             KeepKeys :
[2023/04/14 13:45:43] ppocr INFO:                 keep_keys : ['image', 'label', 'length']
[2023/04/14 13:45:43] ppocr INFO:     loader :
[2023/04/14 13:45:43] ppocr INFO:         batch_size_per_card : 256
[2023/04/14 13:45:43] ppocr INFO:         drop_last : True
[2023/04/14 13:45:43] ppocr INFO:         num_workers : 1
[2023/04/14 13:45:43] ppocr INFO:         shuffle : True
[2023/04/14 13:45:43] ppocr INFO: profiler_options : None
[2023/04/14 13:45:43] ppocr INFO: train with paddle 2.4.1 and device Place(gpu:0)
[2023/04/14 13:45:43] ppocr INFO: Initialize indexs of datasets:['./data/rec_gt_train.txt']
[2023/04/14 13:45:43] ppocr INFO: Initialize indexs of datasets:['./data/rec_gt_test.txt']
W0414 13:45:43.848752 12972 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 12.0, Runtime API Version: 11.6
W0414 13:45:43.852738 12972 gpu_resources.cc:91] device: 0, cuDNN Version: 8.4.
[2023/04/14 13:45:44] ppocr INFO: train dataloader has 63 iters
[2023/04/14 13:45:44] ppocr INFO: valid dataloader has 17 iters
[2023/04/14 13:45:45] ppocr WARNING: The shape of model params head.fc.bias [1211] not matched with loaded params head.fc.bias [37] !
[2023/04/14 13:45:45] ppocr WARNING: The shape of model params head.fc.weight [512, 1211] not matched with loaded params head.fc.weight [512, 37] !
[2023/04/14 13:45:45] ppocr INFO: load pretrain successful from ./rec_r34_vd_none_bilstm_ctc_v2.0_train/best_accuracy
[2023/04/14 13:45:45] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations

(py38) E:\Backup\Desktop\手写字体检测\PaddleOCR-2.6.0>

- 完整报错/Complete Error Message：
没有报错信息 运行到最后 就自动退出了

**Comment by smallhaozigithub** [comment]
今天碰到同样的问题，不过是训练re的时候，也是加载完配置与数据后就自动退出了，没有报错信息。

**Comment by smallhaozigithub** [comment]
W0414 12:47:49.153649 17891 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 11.2, Runtime API Version: 11.2
W0414 12:47:49.157733 17891 gpu_resources.cc:91] device: 0, cuDNN Version: 8.2.
[2023/04/14 12:47:54] ppocr INFO: train dataloader has 8 iters
[2023/04/14 12:47:54] ppocr INFO: valid dataloader has 8 iters
[2023/04/14 12:47:54] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 19 iterations

**Comment by liuzimo** [comment]
我感觉肯定里面什么地方 try 了 然后把报错信息吃了 直接退出了

**Comment by Go4Lose** [comment]
我也是一样的情况

**Comment by an1018** [comment]
num workers设置0 再试试

**Comment by smallhaozigithub** [comment]
设置成了0也是一样

**Comment by Go4Lose** [comment]
batch_size_per_card 从4开始往上拉试试？

**Comment by liuzimo** [comment]
> 

这个有效果
